packages/api/test/functional/service.spec.js

```diff
@@ -175,3 +175,24 @@ test('DELETE /services/:id deletes a service', async ({ client, assert }) => {
 		AlgoliaSearch.initIndex().deleteObject.withArgs(service.toJSON().objectID).calledOnce,
 	);
 });
+
+test('GET /services/my-services get authenticated user services', async ({ client, assert }) => {
+	const { user } = await createUser({ append: { status: 'verified' } });
+
+	await Promise.all(
+		[1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15].map(() =>
+			Factory.model('App/Models/Service').create({
+				user_id: user.id,
+			}),
+		),
+	);
+
+	const response = await client
+		.get(`/services/my-services`)
+		.loginVia(user, 'jwt')
+		.end();
+
+	assert.equal(response.body.length, 10);
```

Comment:
Any changes in the default number of items per page would break this test 😬 
```suggestion
	const perPage = 10;

	const response = await client
		.get(`/services/my-services`)
		.send({ embed: true, perPage })
		.loginVia(user, 'jwt')
		.end();

	assert.equal(response.body.length, perPage);
```